src/main/java/org/openstreetmap/atlas/checks/flag/FlaggedPoint.java

```diff
@@ -36,6 +51,30 @@ public FlaggedPoint(final Location point)
     @Override
     public Map<String, String> getProperties()
     {
-        return Collections.EMPTY_MAP;
+        return this.properties;
+    }
+
+    @SuppressWarnings("unchecked")
+    private Map<String, String> initProperties(final LocationItem locationItem)
+    {
+        try
+        {
+            final Map<String, String> tags = locationItem.getTags();
+            tags.put(ITEM_IDENTIFIER_TAG, locationItem.getIdentifier() + "");
+            tags.put(OSM_IDENTIFIER_TAG, locationItem.getOsmIdentifier() + "");
+            if (locationItem instanceof Node)
+            {
+                tags.put(ITEM_TYPE_TAG, NODE_TAG);
+            }
+            else if (locationItem instanceof Point)
+            {
+                tags.put(ITEM_TYPE_TAG, POINT_TAG);
+            }
+            return tags;
+        }
+        catch (final Exception exception)
+        {
+            return Collections.EMPTY_MAP;
```

Comment:
This seems odd, what else could the `LocationItem` be besides a `Node` or `Point`? At minimum, I'd log if this ever happens, but throwing might be an option too.